### ISSUE TITLE
Improvements on PlantUML output

### ISSUE THREAD

**Issue by enikao** [LionWeb-io/lionweb-typescript#127]
* Leaving out `Node` as superclass would make diagrams more readable (less lines).
* In typescript an optional property is shown with a question mark postfix. Using this instead of <> might make the diagrams easier to read (less space needed).

Extracted from https://github.com/LionWeb-io/lionweb-mps/pull/89#issuecomment-1938382248

**Comment by enikao** [comment]
Wouldn't something like

```
if (concept.extends == Node) {
  // don't write `extends Node`
}
```

suffice?

**Comment by dslmeinte** [comment]
Isn't it up to the caller of the functionality to not do that? It's not required to specify that a classifier extends `Node` if it doesn't any other.

OTOH: it makes sense to weed it out.

**Comment by enikao** [comment]
We _could_ make it a configuration flag, but I can see very few use cases where it makes sense to display the inheritance from `Node`.

**Comment by joswarmer** [comment]
If you really want to support this and more, you need something like the MPOS diagram language that I created for the Tax Office and Clario. But that would be a serious extension.

**Comment by dslmeinte** [comment]
> If you really want to support this and more, you need something like the MPS diagram language that I created for the Tax Office and Clario. But that would be a serious extension.

Exactly, and as always it's about the balance. This functionality is really meant as something “quick-and-not-entirely-dirty” to gain insight, but not as a tool to make well-documented diagrams for knowledge transfer with.

**Comment by dslmeinte** [comment]
Fixed by #128
